containerz/containerz.proto

```diff
@@ -54,12 +54,10 @@ service Containerz {
   rpc RemoveImage(RemoveImageRequest)
     returns (RemoveImageResponse) {}
 
-  // RemoveContainer deletes images that match the spec defined in the
-  // request. If the image is associated to a running container then an error
-  // is returned. If the specified container image does not exist, this
-  // operation is a no-op.
-  //
-  // Deprecated - use RemoveImage instead.
+  // RemoveContainer deletes the container that matches the instance
+  // name. If the specified container does not exist, this
+  // operation is a no-op. The container should not be removed if it is
+  // running unless force is specified in the request.
```

Comment:
We should probably update the PR description to say that the container should not be removed if it's running -- I came to look here since the description says something like "`RemoveContainer` should not remove a container"!

Reply:
Oh oops - I dropped the `not`